### DISCUSSION TITLE
Allow for Pasting in Poster Images from Clipboard

### DISCUSSION THREAD

**Discussion by kanjieater** [gotson/komga#1597] [Ideas]
### Describe your suggested feature

In this area I'd like to be able to paste from clipboard an image, and have Komga set it to the main poster:
![image](https://user-images.githubusercontent.com/32607317/235822379-156013c0-de35-4c83-b5b3-c47021c22e91.png)
I use https://github.com/edwinbadillo/KoMI to copy cover images. The current process is, I have to save them to disk, then drag and drop them. I was hoping to be able to just copy the image and paste it like other programs such as musicbee &  calibre allow (although these are native apps, not running in a browser). Thank you for the consideration.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
@kanjieater is this still relevant ?

I am looking at how clipboard works, and depending on the scenario this may or may not be possible.

For instance, tested on a mac:
- when copying a file from the finder/explorer, the clipboard only contains a file path, not the actual image. Browser security forbids accessing files, so this cannot work.
- when taking a screenshot, an image is available and can be pasted
- when using the "Copy image" from a browser, an image is available and can be pasted

**Reply by gotson** [reply]
well, i managed to work around it by using the paste event from CTRL/CMD+V, it works fine

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.27.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.27.0))
